### ISSUE TITLE
Field inside of postbox impossible to see or edit in page editing

### ISSUE THREAD

**Issue by Crmy7** [htmlburger/carbon-fields#1248]
## Version

  - Carbon Fields: 3.3
  - WordPress: 6.6.1
  - PHP: 7.4.33 64bit
 

## Expected Behavior
I want to edit the custom fields on my pages but I can't open the boxes to edit.

## Actual Behavior
I want to edit the custom fields on my pages but I can't open the boxes to edit.

## Errors:

Uncaught ReferenceError: regeneratorRuntime is not defined
    at vendor.min.js?ver=3.3.0:57:33240
    at Module.<anonymous> (vendor.min.js?ver=3.3.0:57:34368)
    at n (vendor.min.js?ver=3.3.0:1:110)
    at vendor.min.js?ver=3.3.0:1:902
    at vendor.min.js?ver=3.3.0:1:912
    at core.min.js?ver=3.3.0:1 

Uncaught TypeError: Cannot read properties of undefined (reading '@wordpress/i18n')
    at core.min.js?ver=3.3.0:1:1495
    at Object.<anonymous> (core.min.js?ver=3.3.0:1:1515)
    at n (core.min.js?ver=3.3.0:1:142)
    at Module.<anonymous> (core.min.js?ver=3.3.0:47:19601)
    at n (core.min.js?ver=3.3.0:1:142)
    at core.min.js?ver=3.3.0:1:941
    at core.min.js?ver=3.3.0:1:952
    at metaboxes.min.js?ver=3.3.0:1 

Uncaught TypeError: Cannot read properties of undefined (reading '@wordpress/i18n')
    at metaboxes.min.js?ver=3.3.0:1:1706
    at Object.<anonymous> (metaboxes.min.js?ver=3.3.0:1:1726)
    at n (metaboxes.min.js?ver=3.3.0:1:147)
    at Module.<anonymous> (metaboxes.min.js?ver=3.3.0:1:35168)
    at n (metaboxes.min.js?ver=3.3.0:1:147)
    at metaboxes.min.js?ver=3.3.0:1:946
    at metaboxes.min.js?ver=3.3.0:1:956

**Comment by denspit** [comment]
I have the same error after updating WordPress to version 6.6.1

**Comment by Crmy7** [comment]
@denspit Were you able to solve the problem?

**Comment by Crmy7** [comment]
@lstoyanoff Do you still have the same problem?

**Comment by denspit** [comment]
@Crmy7  No. The plugin has not been updated, nothing has changed.

**Comment by Crmy7** [comment]
Is it the WordPress version change? @denspit

**Comment by quimbyosis** [comment]
Chiming in to say I'm having a similar issue - when attempting to make any edits to certain custom blocks, there is a message saying "Your site doesn't include support for the 'carbon-fields/xyz' block. You can leave this block intact or remove it entirely."

**Comment by tempranova** [comment]
Hey guys, I was running into this, found a fix. The issue is basically that regenerator-runtime was a script that used to be a dependency for wp-polyfill, but it's removed. It's referenced in this WP update: https://make.wordpress.org/core/2021/06/28/miscellaneous-developer-focused-changes-in-wordpress-5-8/ that they might remove it one day.

Anyway, to do the fix, go to /vendor/htmlburger/carbon-fields/core/Loader/Loader.php, and line 218 or so (I've done some other modifications so it may not be exact, you'll see this):

```
$this->enqueue_script( 'vendor', array( 'wp-polyfill', 'jquery' ) );
$this->enqueue_script( 'core', array( 'carbon-fields-vendor' ) );
$this->enqueue_script( 'metaboxes', array( 'carbon-fields-vendor', 'carbon-fields-core' ) );
```

before the first line (the one enqueing wp-polyfill), add this:

```
$this->enqueue_script( 'vendor', array( 'regenerator-runtime', 'jquery' ) );
```

And presto, the fields should appear again.

**Comment by Crmy7** [comment]
Hi, I ran the command again to regenerate the vendor and it solved the problem, the postboxes reappeared.

`composer require htmlburger/carbon-fields`

Thanks for your help.

**Comment by steskey-clickup** [comment]
> before the first line (the one enqueing wp-polyfill), add this:
> 
> $this->enqueue_script( 'vendor', array( 'regenerator-runtime', 'jquery' ) );
> And presto, the fields should appear again.

Thanks! @tempranova 

Didn't want to comment on a closed thread, but should anyone find it:
- Updating to the latest version via Composer did not fix the issue for my instance (v3.6.5 at time of writing)
- Enqueuing the `regenerator-runtime` DID fix the issue
